plugins/inputs/mongodb/mongodb_data.go

```diff
@@ -238,7 +238,7 @@ var defaultTCMallocStats = map[string]string{
 	"tcmalloc_spinlock_total_delay_ns":          "TCMallocSpinLockTotalDelayNanos",
 	"tcmalloc_pageheap_free_bytes":              "TCMallocPageheapFreeBytes",
 	"tcmalloc_pageheap_unmapped_bytes":          "TCMallocPageheapUnmappedBytes",
-	"tcmalloc_pageheap_committed_bytes":         "TCMallocPageheapComittedBytes",
+	"tcmalloc_pageheap_committed_bytes":         "TCMallocPageheapCommittedBytes",
```

Comment:
Hmmm are we sure that the type is **not** in the output of MongoDB stats? Otherwise this will break the plugin...